release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,345 @@
+                    ],
+                    "unsupported-versions": [
+                        "3.16"
+                    ]
+                },
+                {
+                    "id": "debian",
+                    "name": "Debian",
+                    "link": "https://www.debian.org/",
+                    "lifecycle": "https://wiki.debian.org/DebianReleases",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "12"
+                    ]
+                },
+                {
+                    "id": "fedora",
+                    "name": "Fedora",
+                    "link": "https://fedoraproject.org/",
+                    "lifecycle": "https://fedoraproject.org/wiki/End_of_life",
+                    "architectures": [
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "40",
+                        "39"
+                    ],
+                    "unsupported-versions": [
+                        "38"
+                    ]
+                },
+                {
+                    "id": "opensuse",
+                    "name": "openSUSE Leap",
+                    "link": "https://www.opensuse.org/",
+                    "lifecycle": "https://en.opensuse.org/Lifetime",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "15.6",
+                        "15.5"
+                    ]
+                },
+                {
+                    "id": "rhel",
+                    "name": "Red Hat Enterprise Linux",
+                    "link": "https://access.redhat.com/",
+                    "lifecycle": "https://access.redhat.com/support/policy/updates/errata/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "9",
+                        "8"
+                    ],
+                    "notes": [
+                        "Red Hat family distributions are supported per [Linux compatibility and support](../../linux-support.md)."
+                    ]
+                },
+                {
+                    "id": "sles",
+                    "name": "SUSE Enterprise Linux",
+                    "link": "https://www.suse.com/",
+                    "lifecycle": "https://www.suse.com/lifecycle/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "15.5"
```

Comment:
SLES still supports `12.5` as well.

Reply:
.NET 8+ cannot run on 12.5, due to requiring glibc 2.23.

From 12.5

```
$ ldd --version
ldd (GNU libc) 2.22
```